### PR TITLE
As user I can navigate between pages #2

### PR DESCRIPTION
**Pull request by tinazavr**

### CODE REVIEW

File: src/app/app.component.html [hunk 1]
```diff
@@ -1,336 +1,16 @@
...
-    <div class="left-side">
-      <svg
-        xmlns="http://www.w3.org/2000/svg"
-        viewBox="0 0 982 239"
-        fill="none"
-        class="angular-logo"
+<mat-sidenav-container>
+  <mat-sidenav opened mode="side" [fixedInViewport]="true">
+    <mat-nav-list>
+      <a mat-list-item routerLink="/categories" routerLinkActive="activebutton"
```
Comment (maksymzav) [inline src/app/app.component.html:4]: for the nav item you seem to style the mat-list-item as a button.
Usually we tend to use existing solutions every time we can. So when our design library implements the needed element: button, we reuse it.
So the code with a button would look like this:
```html
<mat-sidenav-container>
  <mat-sidenav opened mode="side" [fixedInViewport]="true">
    <mat-nav-list>
      <mat-list-item routerLink="/categories" routerLinkActive="activebutton">
        <a mat-button>Categories</a>
      </mat-list-item>
...
```
After you have a button, you don't need to write all styles from scratch. You can only add the needed styles that are missing, if there are any.

Comment (maksymzav) [reply src/app/app.component.html:4]: Another thing is how you track the active link: 
```html
<a mat-list-item routerLink="/categories" routerLinkActive="activebutton"
```

the `routerLinkActive` directive adds the `activebutton` class to a current active nav item.
Material library has a built-in way to specify the active button.
You can use the `color` attribute for that.
so if the normal button looks like 
```angular2html
<a mat-button color="primary">My button</a>
```
the active one can look like this:
```angular2html
<a mat-button color="accent">My active button</a>
```
You can use it in your code:
```angular2html
<mat-list-item routerLink="/categories" routerLinkActive #routerLink="routerLinkActive">
        <a mat-button [color]="routerLink.isActive ? 'accent' : 'primary'">Categories</a>
      </mat-list-item>
```

Pay attention for `#routerLink="routerLinkActive"`. It create a template variable routerLink, and the API of routerLinkActive directive is available using this variable. Using it, you can write conditions that depends whether the link is active: `[color]="routerLink.isActive ? 'accent' : 'primary'"`

File: src/app/app.routes.ts [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+import { CategoriesComponent } from './categories/categories.component';
+import { SpendingsComponent } from './spendings/spendings.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  { path: '', redirectTo: '/spendings', pathMatch: 'full' },
+
+  { path: 'categories', component: CategoriesComponent },
+  { path: 'spendings', component: SpendingsComponent },
+  //{ path: '**', component: PageNotFoundComponent },
```
Comment (maksymzav) [inline src/app/app.routes.ts:10]: please remove all commented code when you submit pull requests

**Review by maksymzav — CHANGES_REQUESTED** [review]

File: src/app/app.component.html (hunk 1 above)
Comment (tinazavr) [reply src/app/app.component.html:4]: > for the nav item you seem to style the mat-list-item as a button. Usually we tend to use existing solutions every time we can. So when our design library implements the needed element: button, we reuse it. So the code with a button would look like this:
> 
> ```
> <mat-sidenav-container>
>   <mat-sidenav opened mode="side" [fixedInViewport]="true">
>     <mat-nav-list>
>       <mat-list-item routerLink="/categories" routerLinkActive="activebutton">
>         <a mat-button>Categories</a>
>       </mat-list-item>
> ...
> ```
> 
> After you have a button, you don't need to write all styles from scratch. You can only add the needed styles that are missing, if there are any.

✅ Done

Comment (tinazavr) [reply src/app/app.component.html:4]: > Another thing is how you track the active link:
> 
> ```
> <a mat-list-item routerLink="/categories" routerLinkActive="activebutton"
> ```
> 
> the `routerLinkActive` directive adds the `activebutton` class to a current active nav item. Material library has a built-in way to specify the active button. You can use the `color` attribute for that. so if the normal button looks like
> 
> ```
> <a mat-button color="primary">My button</a>
> ```
> 
> the active one can look like this:
> 
> ```
> <a mat-button color="accent">My active button</a>
> ```
> 
> You can use it in your code:
> 
> ```
> <mat-list-item routerLink="/categories" routerLinkActive #routerLink="routerLinkActive">
>         <a mat-button [color]="routerLink.isActive ? 'accent' : 'primary'">Categories</a>
>       </mat-list-item>
> ```
> 
> Pay attention for `#routerLink="routerLinkActive"`. It create a template variable routerLink, and the API of routerLinkActive directive is available using this variable. Using it, you can write conditions that depends whether the link is active: `[color]="routerLink.isActive ? 'accent' : 'primary'"`

✅ Done

File: src/app/app.routes.ts (hunk 2 above)
Comment (tinazavr) [reply src/app/app.routes.ts:10]: > please remove all commented code when you submit pull requests

✅ Done

**Review by maksymzav — APPROVED** [review]

**Pull request merged**